### PR TITLE
RSDK-9295 - refactor disable-web-browser name and behavior

### PR DESCRIPTION
**Pull request by stuqdog**
Adds a slightly less verbose `no-browser` alias to the `disable-web-browser` flag, and changes default behavior such that a command won't fail when a browser can't be opened, but rather inform the user that the browser failed to open and that they can still visit the provided URL to log in.

Tested manually.
Without flag, artificially injected error at browser open attempt:
Screenshot 2024-12-23 at 14 08 17

With flag:
Screenshot 2024-12-23 at 14 20 36

### CODE REVIEW

**Comment by stuqdog** [comment]
@thegreatco fyi

File: cli/auth.go [hunk 1]
```diff
@@ -660,7 +660,8 @@ func newCLIAuthFlowWithAuthDomain(authDomain, audience, clientID string, console
 	}
 }
 
-func (a *authFlow) loginAsUser(ctx context.Context) (*token, error) {
+func (a *authFlow) loginAsUser(c *cli.Context) (*token, error) {
+	ctx := c.Context
```
Comment (purplenicole730) [inline cli/auth.go:664]: For my own knowledge more than anything, why the change?

**Review by purplenicole730 — APPROVED** [review]

File: cli/auth.go (hunk 1 above)
Comment (stuqdog) [reply cli/auth.go:664]: Good question! The `cli.Context` has an `io.Writer` on it that allows us to write warning messages to terminal when connecting to the browser fails.
